MachObfuscator.xcodeproj/xcshareddata/xcschemes/MachObfuscator.xcscheme

```diff
@@ -0,0 +1,115 @@
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "D48B8F431F7FEBC200E73BBD"
+               BuildableName = "MachObfuscator"
+               BlueprintName = "MachObfuscator"
+               ReferencedContainer = "container:MachObfuscator.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+      </BuildActionEntries>
+   </BuildAction>
+   <TestAction
+      buildConfiguration = "Debug"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
+      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      shouldUseLaunchSchemeArgsEnv = "YES">
+      <Testables>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "D4F792571FD74E4800EE6EBE"
+               BuildableName = "MachObfuscatorTests.xctest"
+               BlueprintName = "MachObfuscatorTests"
+               ReferencedContainer = "container:MachObfuscator.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
+      </Testables>
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "D48B8F431F7FEBC200E73BBD"
+            BuildableName = "MachObfuscator"
+            BlueprintName = "MachObfuscator"
+            ReferencedContainer = "container:MachObfuscator.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
+      <AdditionalOptions>
+      </AdditionalOptions>
+   </TestAction>
+   <LaunchAction
+      buildConfiguration = "Debug"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
+      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      launchStyle = "0"
+      useCustomWorkingDirectory = "NO"
+      ignoresPersistentStateOnLaunch = "NO"
+      debugDocumentVersioning = "YES"
+      debugServiceExtension = "internal"
+      allowLocationSimulation = "YES">
+      <BuildableProductRunnable
+         runnableDebuggingMode = "0">
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "D48B8F431F7FEBC200E73BBD"
+            BuildableName = "MachObfuscator"
+            BlueprintName = "MachObfuscator"
+            ReferencedContainer = "container:MachObfuscator.xcodeproj">
+         </BuildableReference>
+      </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-v"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-m realWords"
+            isEnabled = "YES">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "/Users/kamilkosowski/Documents/OSXDemo-kopia/OSXDemo.app"
```

Comment:
Your local path here. Do you think we need a shared schema at all? Each one of us would use different parameters anyway.